features/admin.userstores.v1/pages/user-stores.tsx

```diff
@@ -222,6 +228,50 @@ const UserStores: FunctionComponent<UserStoresPageInterface> = (
         setFilteredUserStores(userStores);
     };
 
+    const addUserDropdownTrigger: ReactElement = (
+        <PrimaryButton
+            data-componentid={ `${ testId }-add-user-button` }
+            data-testid={ `${ testId }-add-user-button` }
```

Comment:
Do we need both `componentid` and `testid` for new components?

Reply:
This is added for backward compatibility.